examples/08_deps_with_different_platforms/README.md

```diff
@@ -0,0 +1,37 @@
+# Example 08: Building deps on different platforms
```

Comment:
Bazel world will celebrate you forever for this example!!! Thank you!

Reply:
(: :tada: